src/apps/crud/table/DefaultCRUDTable.jsx

```diff
@@ -112,6 +180,55 @@ const useOnMount = (props) => {
   return { ref };
 };
 
+const CustomizedMenus = (props) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const handleMenuClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = (action) => {
+    setAnchorEl(null);
+  };
+  const handleActionClick = (action, item) => {
+    setAnchorEl(null);
+    if (props.onOtherAction != null) {
+      props.onOtherAction(action, item);
+    }
+  };
+  return (
+    <div>
+      <Button
+        id="demo-customized-button"
+        aria-controls={open ? 'demo-customized-menu' : undefined}
```

Comment:
ARIA props are only used for screen readers for accessibility reasons. Not really needed

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes